miqa/core/models/project.py

```diff
@@ -77,6 +77,24 @@ def get_user_role(self, user):
             key=lambda perm: perm_order.index(perm) if perm in perm_order else -1,
         )[-1]
 
+    def get_status(self):
+        tier_2_reviewers = [
+            user.id for user in get_users_with_perms(self, only_with_perms_in=['tier_2_reviewer'])
+        ]
+        scans_in_project = Scan.objects.filter(
+            experiment__in=self.experiments.all()
+        ).prefetch_related('decisions')
+        complete_scans_in_project = [
+            scan
+            for scan in scans_in_project
+            if scan.decisions.count() > 0
+            and scan.decisions.latest('created').creator.id in tier_2_reviewers
+        ]
```

Comment:
FWIW, this filter could also be expressed as part of the initial query, but I don't want that to hold up this PR. It would be a good learning exercise, though!